sorc/ncep_post.fd/MDL2P.f

```diff
@@ -1816,6 +1819,65 @@ SUBROUTINE MDL2P(iostatusD3D)
           ENDIF
         ENDIF
 !     
+!***  CHIPSI
+!
+IF ( (IGET(1021) > 0 .or. IGET(1022) > 0) .and. MODELNAME == 'GFS' ) THEN
+          IF (LVLS(LP,IGET(1021)) > 0 .or. LVLS(LP,IGET(1022)) > 0) THEN
```

Comment:
@KarinaAsmar-NOAA This if statement might not be needed.

Reply:
@KarinaAsmar-NOAA I did consider about the vertical levels. Sorry for the misleading comments.